packages/esm-patient-registration-app/src/patient-registration/field/field.scss

```diff
@@ -92,6 +103,10 @@
   margin-bottom: spacing.$spacing-05;
 }
 
+.radioButton {
+  padding-block: spacing.$spacing-03 !important;
```

Comment:
padding-block? I don't know that, happy to learn

Reply:
Don't use `!important`

Reply:
> padding-block? I don't know that, happy to learn

top and bottom padding